### DISCUSSION TITLE
The Lingua Libre Signit

### DISCUSSION THREAD

**Discussion by Clin-Tech** [lingua-libre/SignIt#62] [General]
This is a nice project concept , and I am interested in working on this under my mentor if allowed. I am knowledgeable in the skills required.

**Comment by hugolpz** [comment]
Hello @97Clinton ,
Thank you for your message.
My apologize I didn't noticed your message earlier we don't use Github `Discussions` much. 
You were interested by the Google Summer of Code 2024 right ?

**Comment by Clin-Tech** [comment]
Yes , I was really interested.
But now I don’t think I’ve a chance anymore since you just got to reply me
